lib/fluent/system_config.rb

```diff
@@ -46,6 +47,7 @@ class SystemConfig
     config_param :strict_config_value, :bool, default: nil
     config_param :enable_msgpack_time_support, :bool, default: nil
     config_param :disable_shared_socket, :bool, default: nil
+    config_param :without_input_metrics, :bool, default: true
```

Comment:
Similar to other parameters, `enable_input_metrics` may be better.
(Even though `without_source` exists)

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3411/commits/334b0490a5f73ce4540835f1d030d512f45c1ebd.